test/cctest/test_inspector_socket.cc

```diff
@@ -10,13 +10,11 @@ static const int MAX_LOOP_ITERATIONS = 10000;
 
 #define SPIN_WHILE(condition)                                                  \
   {                                                                            \
-    bool timed_out = false;                                                    \
-    uv_timer_t* timer = start_timer(&timed_out);                               \
-    while (((condition)) && !timed_out) {                                      \
+    Timeout timeout(&loop);                                                    \
+    while (((condition)) && !timeout.timed_out) {                              \
```

Comment:
While you're here, maybe you can remove the extraneous parentheses around `condition`?